__tests__/integration-metrics.test.js

```diff
@@ -73,6 +74,11 @@ describe('Metrics SDK Integration Tests', () => {
     // Annoyingly this works under macOS, so it must be a platform
     // difference when running under docker/linux.
     const [command, ...args] = process.env.EXAMPLE_SERVER.split(' ');
+    if (command === 'php') {
+      // Laravel's `artisan serve` command doesn't pick up `PORT` environmental variables, instead
+      // requiring that they're supplied as a command line argument.
+      args.push(`--port=${PORT}`);
```

Comment:
I'm going to keep this arg addition in place instead of forking passing either `PORT` or `SERVER_PORT` to the command. This definitely feels more explicit and easier to understand.